RPi4 make process throws alignment error

Following the "quick start" section of the README, generates an error and make failure:
```
Scanning dependencies of target msg
[ 83%] Building C object msg/CMakeFiles/msg.dir/src/cfe_msg_ccsdspri.c.o
[ 83%] Building C object msg/CMakeFiles/msg.dir/src/cfe_msg_init.c.o
[ 84%] Building C object msg/CMakeFiles/msg.dir/src/cfe_msg_msgid_shared.c.o
[ 85%] Building C object msg/CMakeFiles/msg.dir/src/cfe_msg_sechdr_checksum.c.o
/home/pi/src/NASA/cFS/cfe/modules/msg/src/cfe_msg_sechdr_checksum.c: In function ‘CFE_MSG_GenerateChecksum’:
/home/pi/src/NASA/cFS/cfe/modules/msg/src/cfe_msg_sechdr_checksum.c:82:49: error: cast increases required alignment of target type [-Werror=cast-align]
     cmd->Sec.Checksum = CFE_MSG_ComputeCheckSum((CFE_MSG_Message_t *)cmd);
                                                 ^
cc1: all warnings being treated as errors
make[7]: *** [msg/CMakeFiles/msg.dir/build.make:102: msg/CMakeFiles/msg.dir/src/cfe_msg_sechdr_checksum.c.o] Error 1
make[6]: *** [CMakeFiles/Makefile2:613: msg/CMakeFiles/msg.dir/all] Error 2
make[5]: *** [Makefile:141: all] Error 2
make[4]: *** [CMakeFiles/native_default_cpu1-all.dir/build.make:57: CMakeFiles/native_default_cpu1-all] Error 2
make[3]: *** [CMakeFiles/Makefile2:298: CMakeFiles/native_default_cpu1-all.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:177: CMakeFiles/mission-all.dir/rule] Error 2
make[1]: *** [Makefile:214: mission-all] Error 2
make: *** [Makefile:120: all] Error 2
pi@renegade:~/src/NASA/cFS $ 
```
At first I thought this was because I was running on the 64-bit Beta Raspberry Pi OS, so I tried it on a different Pi4, this one running the stable 32-bit distribution (Buster, latest release (8/20), updated yesterday (8/30)). It fails on the 32-bit version also.

--- a/modules/msg/src/cfe_msg_sechdr_checksum.c
+++ b/modules/msg/src/cfe_msg_sechdr_checksum.c
@@ -78,8 +78,9 @@ int32 CFE_MSG_GenerateChecksum(CFE_MSG_Message_t *MsgPtr)
     /* Zero checksum so new checksum will be correct */
     cmd->Sec.Checksum = 0;
 
-    /* Compute and set */
-    cmd->Sec.Checksum = CFE_MSG_ComputeCheckSum((CFE_MSG_Message_t *)cmd);
+    /* Compute using aligned MsgPtr and set, suppress false style warning */
+    /* cppcheck-suppress redundantAssignment */
+    cmd->Sec.Checksum = CFE_MSG_ComputeCheckSum(MsgPtr);
 
     return CFE_SUCCESS;
 }

